src/storage/postgres/src/main/java/org/locationtech/geogig/storage/postgresql/DataSourceManager.java

```diff
@@ -64,6 +160,8 @@ protected DataSource connect(Environment.ConnectionConfig config) {
         hc.setPassword(config.getPassword());
         hc.setConnectionTimeout(5000);
 
+        String jdbcUrl = config.getServer() + ":" + config.getPortNumber() + "/"
```

Comment:
It's used a couple more times and since it's not in a loop or anything (will be done just once per connection, so 10 times if you fill up the connection pool). So didn't think it's worth the extra complexity of adding three conditionals and initializing it to null if the logger is not configured to the debug level just to avoid a simple string concatenation.